theano/tensor/elemwise.py

```diff
@@ -185,6 +206,13 @@ def __init__(self, input_broadcastable, new_order, inplace=True):
         if self.inplace:
             self.view_map = {0: [0]}
```

Comment:
No, because this doesn't actually destroy anything.  I am saying that we can probably remove the inplace property and act as if it was always True.